api/src/routes/anime/animeRoutes.ts

```diff
@@ -0,0 +1,15 @@
+import express from "express";
+import { animeController } from "@@root/controllers";
+import { extractPaginationParams, validation } from "@@middleware/index.js";
+
+const animeRoutes = express.Router();
+
+animeRoutes.route("/").get(extractPaginationParams, animeController.index);
+
+animeRoutes.route("/anime/id/:id")
```

Comment:
Can you remove the `/anime` portion of this url, otherwise the route will be `/anime/anime/id/<some id value>`